src/Compilers/CSharp/Test/Symbol/DocumentationComments/CrefTests.cs

```diff
@@ -6703,11 +6703,205 @@ void S()
 
             var parameter = cref.Parameters.Parameters.Single();
             Assert.Equal(SyntaxKind.InKeyword, parameter.RefKindKeyword.Kind());
+            Assert.Equal(SyntaxKind.None, parameter.ReadOnlyKeyword.Kind());
 
             var parameterSymbol = ((IMethodSymbol)model.GetSymbolInfo(cref).Symbol).Parameters.Single();
             Assert.Equal(RefKind.In, parameterSymbol.RefKind);
         }
 
+        [Fact]
+        public void CRef_RefReadonlyParameter()
+        {
+            var source = """
+                class Test
+                {
+                    void M(ref readonly int x)
+                    {
+                    }
+
+                    /// <summary>
+                    /// <see cref="M(ref readonly int)"/>
+                    /// </summary>
+                    void S()
+                    {
+                    }
+                }
+                """;
+
+            verify(CreateCompilation(source, parseOptions: TestOptions.Regular11.WithDocumentationMode(DocumentationMode.Diagnose)).VerifyDiagnostics(
+                // (3,16): error CS8652: The feature 'ref readonly parameters' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
+                //     void M(ref readonly int x)
+                Diagnostic(ErrorCode.ERR_FeatureInPreview, "readonly").WithArguments("ref readonly parameters").WithLocation(3, 16),
+                // (8,26): warning CS1658: The feature 'ref readonly parameters' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.. See also error CS8652.
```

Comment:
not related to this PR: I'm curious whether this warning override will trigger the IDE fixer to turn on the preview LangVer. This would be a pre-existing issue. May need to file an issue

Reply:
It won't. Filed https://github.com/dotnet/roslyn/issues/69228.